doc/man1/s_time.pod

```diff
@@ -13,14 +13,13 @@ B<openssl> B<s_time>
 [B<-cert filename>]
 [B<-key filename>]
 [B<-CApath directory>]
-[B<-CAfile filename>]
+[B<-cafile filename>]
```

Comment:
Because, sadly and stupidly, that is what the flags are and now we have to live with it.